frontend/src/component/FeedItem/FeedItemWithPreview.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import FeedItemHeader from './FeedItemHeader.jsx'
+import FeedItemFooter from './FeedItemFooter.jsx'
+import Preview from './Preview.jsx'
+
+export const FeedItemWithPreview = props => {
+  return (
+    <div>
+      <FeedItemHeader
+        breadcrumbsList={props.breadcrumbsList}
+        content={props.content}
+        eventList={props.eventList}
+        lastModification={props.lastModification}
+        lastModificationEntityType={props.lastModificationEntityType}
+        lastModificationSubEntityType={props.lastModificationSubEntityType}
+        lastModifier={props.lastModifier}
+        modifiedDate={props.modifiedDate}
+        onClickCopyLink={props.onClickCopyLink}
+        onEventClicked={props.onEventClicked}
+        workspaceId={props.workspaceId}
+      />
+      <Preview content={props.content} />
+      <FeedItemFooter
+        commentList={props.commentList}
+        content={props.content}
+        reactionList={props.reactionList}
+      />
+      {props.children}
```

Comment:
You are not using this, is it for later use?

Reply:
Yes, at issues description: _The component ContentWithPreviewActivity_ (a.k.a. FeedItemWithPreview) _should allow children to add the timeline on it_

Reply:
ok, good